### DISCUSSION TITLE
Stuck on "You are more likely to be verified if you have completed your GitHub user profile."

### DISCUSSION THREAD

**Discussion by AlphaHasher** [community/community#30138] [GitHub Education] [answered]
I have filled out every single spot on my GitHub user profile and it still won't allow me to continue with my application because:
`You are more likely to be verified if you have completed your GitHub user profile.`

**Comment by AlphaHasher** [comment, marked as answer]
### Update: I got accepted by getting my hands on my student ID. It looks like what GitHub wants is a VERY official-looking document, or in my case, a dated ID card.

Please continue to post here as others are still struggling with this issue. Have a wonderful day wherever you may be :)

**Reply by AlifPapp** [reply]
I had this issue, but I got it working by doing it through my phone. 
Don't upload a file, take a picture instead.

**Reply by vadrif-draco** [reply]
I don't think the process is complaining about the picture as it is outlining the 'incomplete' GitHub user profile as the main culprit... Or more specifically, it _**should not**_ be the case, otherwise that's just intended misdirection on GitHub's part lol.

**Reply by sixgod-666-jpg** [reply]
Use the transcript!!    I used my transcript and it passed in seconds

**Reply by janawal-wgu** [reply]
My profile is configured as requested, I'm uploading a screen shot of my bursar invoice, one of the options in the list. I get this error.

**Reply by anthonyweidai** [reply]
I got verified just using iphone taking photos. But that's not enough. I failed once by this way. I then added my full name in bio and user name. I also filled in every blank in profile. Then it works within 30s.

**Reply by gdanielbarros** [reply]
No meu caso, apenas envie comprovante de vinculo da faculdade que estou

**Reply by piro-shiki-engineer** [reply]
I had the same problem.  But, I resolve this by using camera of phone. Try it out !!

**Reply by colourfulspring** [reply]
Thanks a lot!!!!!!!!! I failed 5 times until trying  your answer!

**Reply by innomaxx** [reply]
> I had this issue, but I got it working by doing it through my phone. Don't upload a file, take a picture instead.

Hello! I tried your approach but still not working. It even doesn't display any errors, after processing it just reloads page without any. Futhermore, the verification page even is unable to start rear camera of my Pixel phone, it succeeded only one in the beginning LMAO. Any ideas?

**Comment by radar-blip** [comment]
Might have something to do with your email address.
Go through your settings (email) and make sure all your school email addresses are added and updated.

**Reply by AlphaHasher** [reply]
I'll try re-adding my school email

**Reply by AlphaHasher** [reply]
Nope, still the same thing, thanks for suggesting though

**Reply by radar-blip** [reply]
Anytime. 
I dont know what else to tell you, lots of people are having that same problem. 
Maybe look through some other discussions.
Good Luck!

**Comment by RinChanNOWWW** [comment]
Same problem.

------

Update: I used my transcript and passed.

**Comment by WolfsCoding** [comment]
Same for me and getting the message i should use a school id but i dont have one

**Reply by AboulFtouh** [reply]
okey what you did
?/

**Comment by opuu** [comment]


**Reply by AlphaHasher** [reply]
it's added already

**Reply by opuu** [reply]
What documents did you upload as proof of your academic status?

**Reply by AlphaHasher** [reply]
I've tried a dated school schedule, enrolment letter and a few more I forget. I don't have an id yet but will have one soon.

**Reply by opuu** [reply]
Try contacting their support.

**Comment by dan-myles** [comment]
Having the same issue here. No matter what I do or submit it is telling me that I have to complete my github user profile....

**Comment by L-Methionine** [comment]
Same issue here....

**Comment by AlifPapp** [comment]
I'm having the same problem as well

**Comment by dingdev** [comment]
same issue...

**Comment by 1PokerMan** [comment]
same issue here...

**Comment by rocxteady** [comment]
Same issue.

**Comment by LEx66** [comment]
you have to add your school email to the account

**Reply by zyzer01** [reply]
I have
Still same error

**Reply by vadrif-draco** [reply]
I don't think this is the problem, I personally have it added, have always had it added there ever since I originally participated in GitHub Education...

**Comment by GWMCwing** [comment]
Possible Fix:
I have just received reply from Github support. The provided support document has to contain the same name as your github profile name, in order to proceed the auto checking. 
Aka, If your github name is not the same as your name shown on id card, then the checking will say your profile is incomplete, as the checking can not find the name on your github profile.
After, the request has been approved or rejected, you can change your name back to whatever you want while keeping your github student features

**Reply by 1PokerMan** [reply]
I changed my nickname to my first and last name and took a photo of my university notebook and my exam permit, I immediately took the test and received a satisfactory email, Thank you

**Reply by Neccolini** [reply]
I changed my profile name as you said but still getting the same error. Should I change my username too?

**Comment by so1ve** [comment]
I chose the student ID card, submitted the admission letter, and passed...

**Comment by submus** [comment]
If you see that issue that means when the image was processed it did not see your name on whatever document you uploaded so to fix this problem you need to upload a document that has a name matching your GitHub profile name and you don't even have to have a bio.

**Comment by dustonmouse** [comment]
While you are taking the picture of your document make sure your hand is visible on the picture. It's work for me.

**Comment by kavalden** [comment]


**Reply by submus** [reply]
https://github.com/community/community/discussions/30138#discussioncomment-4370943

**Reply by submus** [reply]
Make sure that the name on the document you uploaded matches the name on your GitHub profile.

**Comment by andrewdimmer** [comment]
Just wanted to add my experience as well to help unblock anyone else who might see this. I was trying to take a picture of my class schedule from the online web portal, since my student ID doesn't have any dates on it. Th issue was that the web portal by default only had my first name on it, and that was the issue. By clicking the (at least on my portal) "Display your personal information button" and it showed my first and last name on the portal, that fixed the issue.

**Comment by tcztzy** [comment]
For non-English students: It seems that the image preprocessing is pretrained for English. My personal experience is that take a photo of your ID card, and then annotate the name in English.

**Reply by happytoenter** [reply]
thanks a lot,it works for me;

**Comment by Forest-Lee** [comment]
Same issiue. I used my transcript and changed my profile name to full name, then it passed.

**Reply by RexWzh** [reply]
It works after lots of tries. Try taking a photo of your GitHub profile and the ID card in the same screen. It works for me finally!

**Reply by didyouexpectthat** [reply]
Thanks for the suggestion, Rex. I attend online school and they don't have ID cards. Their verification is from the National Clearinghouse. I ended up half-screening the document with my dates and github profile page and after 17 attempts, I finally got it!
